services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/team/TeamManagerImplTest.java

```diff
@@ -771,6 +770,31 @@ public void testGetIconURL() {
 		
 		assertEquals(expectedUrl, url);
 	}
+
+	@Test
+	public void testGetIconPreviewURL() {
+		String iconFileHandleId = "101";
+
+		Team team = createTeam(null, "name", "description", null, iconFileHandleId, null, null, null, null);
+		when(mockTeamDAO.get(TEAM_ID)).thenReturn(team);
+
+		String iconPreviewFileHandleId = "102";
+
+		when(mockFileHandleManager.getPreviewFileHandleId(iconFileHandleId)).thenReturn(iconPreviewFileHandleId);
+
+		FileHandleUrlRequest urlRequest = new FileHandleUrlRequest(userInfo, iconPreviewFileHandleId)
+				.withAssociation(FileHandleAssociateType.TeamAttachment, TEAM_ID);
+
+		String expectedUrl = "https://testurl.org";
+
+		when(mockFileHandleManager.getRedirectURLForFileHandle(eq(urlRequest))).thenReturn(expectedUrl);
```

Comment:
We can use any() instead of eq(urlRequest) as we are verifying below that the argument is correct.